ui/src/lib/gql/document/useQueries.ts

```diff
@@ -82,9 +100,30 @@ export const useQueries = () => {
     },
   });
 
+  const previewSnapshot = useMutation({
```

Comment:
Please remember to consistently name things. Should be usePreviewSnapshot.

Also, the theme of "do we need projectsnapshots and previewsnapshots" continues.